libexec/rtld-elf/rtld_c18n.c

```diff
@@ -859,15 +858,16 @@ resolve_untrusted_stk_impl(stk_table_index index)
 /*
  * Stack unwinding
  */
-/*
- * Assembly functions that are tail-called when compartmentalisation is
- * disabled.
- */
-uintptr_t _rtld_unw_getcontext_epilogue(uintptr_t, void **);
-struct jmp_args _rtld_unw_setcontext_epilogue(struct jmp_args, void *, void **);
+uintptr_t c18n_get_trusted_stk(uintptr_t, struct trusted_frame **);
```

Comment:
These functions are meant to be used by things calling into rtld for specific functionality, so it might make sense to document that intent here in a comment?